src/pfs/pfs.proto

```diff
@@ -197,6 +197,8 @@ message CommitInfo {
   google.protobuf.Timestamp finishing = 7;
   google.protobuf.Timestamp finished = 8;
   repeated Commit direct_provenance = 13;
+  repeated Commit direct_subvenance = 15;
```

Comment:
did you run the python proto generation? I would expect this change to result in a diff within `python-sdk/api/pfs/__init__.py`

Reply:
Sorry I couldn't get the make proto target to work anymore so I generated the protos locally. Do we know how to run the python proto generation?

Reply:
Okay, I fixed the generation on my end, the files should be there now